lib/src/authenticate.tsx

```diff
@@ -87,14 +102,14 @@ const AuthProvider: FunctionComponent<{ children: any; config: any; }> = ({ chil
 
         AuthClient.init(config);
         setConfigState(config);
-    }, [ config ]);
+    }, [config]);
 
     /**
      * Render state and special case actions
      */
     return (
         <AuthContext.Provider
-            value={ {
+            value={{
```

Comment:
Let's add the `space` back.
```suggestion
            value={ {
```